src/main/scala/net/snowflake/spark/snowflake/FilterPushdown.scala

```diff
@@ -18,7 +18,7 @@
 package net.snowflake.spark.snowflake
 
 import java.sql.{Date, Timestamp}
-
+import scala.language.postfixOps
```

Comment:
what is it?

Reply:
Is it supported by both Scala 2.12 and 2.13?

Reply:
Scala 2.13 needs to import `scala.language.postfixOps` explicitly if postfix operator is used such as `!`.
```
    // clean up download dir
    import sys.process._
    "rm -fr /tmp/test_move_data" !
```

Reply:
and yet. It is supported for both.